entrypoint.sh

```diff
@@ -47,13 +59,34 @@ function split_certs() {
     fi
   done
 
-  mkdir -p /etc/haproxy/certs > /dev/null 2>&1
   rm /etc/haproxy/certs/cert*_gen.pem > /dev/null 2>&1 || true
   mv cert*_gen.pem /etc/haproxy/certs/
-  etcd_set "certs/bundle" "value=true"
   rm cert*_gen.pem > /dev/null 2>&1 || true
 }
 
+HAVE_CERTS=
+
+function load_certs() {
+  mkdir -p /tmp/ssl_certs
+  mkdir -p /etc/haproxy/certs
+
+  if [ -n "${SSL_CERTS}" ]; then
+    echo "[kontena-lb] splitting bundled certificates from SSL_CERTS..."
+    split_certs
+    HAVE_CERTS=true
+  fi
+
+  for certenv in "${!SSL_CERT_@}"; do
```

Comment:
ATM this only supports `${!SSL_CERT_*}` env names that are also valid bash variable names. Env names like `SSL_CERT_test-1` or `SSL_CERT_example.com` do not work...

Reply:
@jakolehm @jnummelin WDYT, does it make sense to restrict the env names to valid variable names? Considering how these `SSL_CERT_*` envs are likely to be generated, it would be convenient to allow names like `SSL_CERT_example.com`. Now it will need some kind of name mangling:

```yaml
    certificates:
      # {% for cert in lb_certs %}
      - subject: {{ cert }}
        name: "SSL_CERT_{{ cert | replace: '.', '_' }}"
        type: env
      # {% endfor %}
```

Reply:
I'm ok with this restriction for now.